nailgun/entities.py

```diff
@@ -8008,3 +8018,22 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
             ignore = set()
         ignore.add('hypervisor_password')
         return super(VirtWhoConfig, self).read(entity, attrs, ignore, params)
+
+    def read_per_organization(self, synchronous=True, **kwargs):
```

Comment:
Rename this to 'get_organization_configs' or simillar.

Reply:
Ok, make sense, thank you